### ISSUE TITLE
Child terms for Abnormal CD4:CD8 ratio

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6252]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Reduced CD4:CD8 ratio 1/1  PMID9096364, OMIM:606367 

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
inverted CD4/CD8 ratio (0.8 ± 0.4; normal range: 0.9–2.6) with normal CD4 counts (1000/μL ± 341; normal range 300–1300/μL) and high CD8 counts (1574/μL ± 855; normal range 300–1300/μL).
https://pubmed.ncbi.nlm.nih.gov/23416241/

**Comment by pnrobinson** [comment]
Also increased CD4:CD8 ratio, 2/2  PMID: 26783323 for 617006 

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4749924/

**Comment by pnrobinson** [comment]
Reduced CD4:CD8 ratio, OMIM:615897  PMID:32161190  2/5 patients

**Comment by pnrobinson** [comment]
also,  reversed a reversed CD4/CD8 ratio of 1:2 as synonym.
https://omim.org/entry/308240
PMID: 9771253  1/1
